tests/unit/output-parser.test.ts

```diff
@@ -0,0 +1,463 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'mocha';
+import { expect } from 'chai';
+import { DeployCloudRunInputs } from '../../src/deploy-cloudrun';
+import { parseUpdateTrafficResponse, parseDeployResponse } from '../../src/output-parser';
+
+const DEFAULT_INPUTS: DeployCloudRunInputs = {
+  image: '',
+  name: '',
+  metadata: '',
+  credentials: '',
+  projectId: '',
+  gcloudVersion: '',
+  envVars: '',
+  secrets: '',
+  region: '',
+  source: '',
+  suffix: '',
+  tag: '',
+  noTraffic: false,
+  revTraffic: '',
+  tagTraffic: '',
+  flags: '',
+};
+
+describe('#output-parser', function () {
+  describe('#parseUpdateTrafficResponse', function () {
+    it('correctly parses update traffic outputs', function () {
```

Comment:
(Can be a follow-up). I've been advocating for table-driven tests: https://mochajs.org/#dynamically-generating-tests

Either the table-driven or closure-driven work pretty well. Example: https://github.com/google-github-actions/actions-utils/blob/main/tests/encoding.test.ts